hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogStateManager.java

```diff
@@ -27,7 +27,8 @@
 import java.util.ArrayList;
 
 /**
- * TODO: javadoc.
+ * DeletedBlockLogStateManager interface to provide BlocksTransaction
```

Comment:
I think we can make this more verbose.

if you trace the code, you'll find it ultimately writes to SCMMetadataStore.getDeletedBlocksTXTable() which says "A Table that keeps the deleted blocks lists and transactions."

You might say it's an interface to manage deleted blocks and record them in the underlying persist store.

Reply:
This seems to be a good proposal, let me revise it.